src/renderers/webgl/WebGLMultiview.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * @author fernandojsg / http://fernandojsg.com
+ */
+
+import { WebGLRenderTarget } from '../WebGLRenderTarget.js';
+import { Matrix3 } from '../../math/Matrix3.js';
+import { Matrix4 } from '../../math/Matrix4.js';
+
+function WebGLMultiview( requested, gl, canvas, extensions, capabilities, properties ) {
+
+	this.isAvailable = function () {
+
+		return capabilities.multiview;
+
+	};
+
+	this.getMaxViews = function () {
+
+		return capabilities.maxMultiviewViews;
+
+	};
+
+	this.isEnabled = function () {
+
+		return requested && this.isAvailable();
+
+	};
+
+	if ( requested && ! this.isAvailable() ) {
+
+		console.warn( 'WebGLRenderer: Multiview requested but not supported by the browser' );
+
+	} else if ( requested !== false && this.isAvailable() ) {
+
+		console.info( 'WebGLRenderer: Multiview enabled' );
+
+	}
+
+	var numViews = 2; // @todo Based on arrayCamera
+	var framebuffer; // multiview framebuffer.
+	var viewFramebuffer; // single views inside the multiview framebuffer.
+	var framebufferWidth = 0;
+	var framebufferHeight = 0;
+
+	var texture = {
+		color: null,
+		depthStencil: null
+	};
+
+	this.computeCameraMatrices = function ( camera ) {
+
+		if ( ! camera.projectionMatrices ) {
+
+			camera.projectionMatrices = new Array( numViews );
+			camera.viewMatrices = new Array( numViews );
+
+			for ( var i = 0; i < numViews; i ++ ) {
+
+				camera.projectionMatrices[ i ] = new Matrix4();
+				camera.viewMatrices[ i ] = new Matrix4();
+
+			}
+
+			if ( camera.isArrayCamera ) {
+
+				for ( var i = 0; i < numViews; i ++ ) {
+
+					camera.projectionMatrices[ i ].copy( camera.cameras[ i ].projectionMatrix );
+					camera.viewMatrices[ i ].copy( camera.cameras[ i ].matrixWorldInverse );
+
+				}
+
+			} else {
+
+				for ( var i = 0; i < numViews; i ++ ) {
+
+					camera.projectionMatrices[ i ].copy( camera.projectionMatrix );
+					camera.viewMatrices[ i ].copy( camera.matrixWorldInverse );
+
+				}
+
+			}
```

Comment:
If I'm right, here L64-L82 should be out of `if ( ! camera.projectionMatrices ) {...}`. Otherwise, `camera.projectionMatrices/viewMatrices` won't be updated.

Reply:
@takahirox yeah! good catch, thanks! :) Updated!